src/components/Text.js

```diff
@@ -0,0 +1,101 @@
+// @flow
+import React from "react";
+import { Text as RnText, StyleSheet } from "react-native";
+import Markdown from "react-native-easy-markdown";
+import type { Node } from "react";
+import type { StyleObj } from "react-native/Libraries/StyleSheet/StyleSheetTypes";
+
+export type TextType =
+  | "h1"
+  | "h2"
+  | "h3"
+  | "h4"
+  | "h5"
+  | "h6"
+  | "text"
+  | "small"
+  | "xSmall";
+
+type Props = {
+  children: Node,
+  type?: TextType,
+  markdown?: boolean,
+  style?: StyleObj
+};
+
+const Text = ({ children, type, markdown, style }: Props) =>
+  markdown ? (
```

Comment:
It looks like there is no functionality overlap between a simple text and a markdown view. The `type` prop is unused when `markdown` is `true`. Do you think it would make sense to split this into 2 separate components `<Text />` and `<MarkdownText />`?

Reply:
> I felt this way might make it clearer that if you're changing the styles, you'll be changing the styles for both text and markdown.

That's a good argument. Agree.

My concern is mainly that the `type` prop is ignored when `markdown` is true. I think it's fine, though.